.stylelintrc.json

```diff
@@ -1,32 +1,32 @@
-{
-  "extends": ["stylelint-config-standard"],
-  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
-  "rules": {
-    "at-rule-no-unknown": [
-      true,
-      {
-        "ignoreAtRules": [
-          "tailwind",
-          "apply",
-          "variants",
-          "responsive",
-          "screen"
-        ]
-      }
-    ],
-    "scss/at-rule-no-unknown": [
-      true,
-      {
-        "ignoreAtRules": [
-          "tailwind",
-          "apply",
-          "variants",
-          "responsive",
-          "screen"
-        ]
-      }
-    ],
-    "csstree/validator": true
-  },
-  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
-}
+{

```

Comment:
- [ ] Kindly update your readme file to a more professional readme instead of using the RoR default template. you can use [this readme template](https://github.com/microverseinc/readme-template) provided by Microverse and please remember to edit each section to better suit the project.